<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iphone_forensics</class>
 <widget class="QMainWindow" name="iphone_forensics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1279</width>
    <height>608</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>맑은 고딕</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>iPhone Forensics Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="folder_select_btn">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="extract_btn">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract</string>
    </property>
   </widget>
   <widget class="QLabel" name="folder_path_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Backup Folder Path</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>761</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txt_result">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>271</width>
      <height>381</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="database_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Database File</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="db_path_select_btn">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="extract_path_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract Path</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="extract_select_btn">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="auto_select_btn">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Select</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="folder_path_line">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>871</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="extract_path_line">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>871</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="db_path_line">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>871</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>490</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>75</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_informaton">
     <item>
      <widget class="QLabel" name="information_label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Infomation</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="iphone_information_btn">
       <property name="text">
        <string>iPhone information</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="backup_information_btn">
       <property name="text">
        <string>Backup Information</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>133</width>
      <height>307</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_artifacts">
     <item>
      <widget class="QLabel" name="artifacts_label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Artifacts</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sms_btn">
       <property name="text">
        <string>SMS Artifact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wallet_pass_btn">
       <property name="text">
        <string>Apple Wallet Artifact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="appleaccount_btn">
       <property name="text">
        <string>Apple Account Artifact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addressbook_btn">
       <property name="text">
        <string>AddressBook Artifact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calendar_btn">
       <property name="text">
        <string>Calendar Artifact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bluetooth_btn">
       <property name="text">
        <string>Bluetooth Artifact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bluetooth_all_btn">
       <property name="text">
        <string>All Bluetooth Artifact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="simcard_btn">
       <property name="text">
        <string>SIM Card Artifact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="application_btn">
       <property name="text">
        <string>Application Artifact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tcc_btn">
       <property name="text">
        <string>TCC Artifact</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="analyze_btn">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyze</string>
    </property>
   </widget>
   <widget class="QLabel" name="processing_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>492</y>
      <width>761</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
